src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -198,6 +200,49 @@ def endAuction():
 
 This example should give you a sense of what Vyper contract syntax is like. For a more detailed description of the functions and variables, [see the docs](https://vyper.readthedocs.io/en/latest/vyper-by-example.html#simple-open-auction).
 
+## Yul and Yul+ {#yul}
+
+**Yul**
+- Intermediate language for Ethereum.
+- Supports EVM and eWASM, and it is designed to be a usable common denominator of both platforms.
+- Good target for high-level optimisation stages that can benefit all target platforms equally.
+
+**Yul+**
+- A low-level, highly efficient extension to Yul.
+- Initially designed for an optimistic rollup contract.
+- Yul+ can be looked at as an experimental upgrade proposal to Yul, adding new features to it.
+
+### Important links {#important-links-2}
+- [Yul Documentation](https://solidity.readthedocs.io/en/latest/yul.html)
+- [Yul+ Documentation](https://github.com/fuellabs/yulp)
+- [Yul+ Playground](https://yulp.fuel.sh/)
+- [Yul+ Introduction Post](https://medium.com/@fuellabs/introducing-yul-a-new-low-level-language-for-ethereum-aa64ce89512f)
+
+### Example contract {#example-contract-2}
```

Comment:
Excuse my ignorance but what does this contract do? We should maybe explain what this simple example does.

Reply:
It's just a super easy power on/off contract that doesn't "do" anything useful, just shows the language. For the more useful stuff the ERC20 contract is interesting, but very long. I don't think we need to explain it in more detail here since the people that Yul is interesting to _should_ be able to get what they need from it. Maybe we should just move the "disclaimer" that I put in the comparison section right on top of the Yul/Yul+ section so that it becomes clear that this is not necessarily for beginners? --> **If you're new to Ethereum and you haven't done any coding with smart contract languages yet, we recommend getting started with Solidity or Vyper and, only once you're familiar with smart contract security best practices and the specifics of working with the EVM, looking into Yul or Yul+.**